src/test/java/com/reckue/post/utils/converters/ConverterTest.java

```diff
@@ -5,18 +5,75 @@
 import com.reckue.post.transfers.TagResponse;
 import org.junit.jupiter.api.Test;
 
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Class ConverterTest allows to test all methods of class Converter.
+ *
+ * @author Kamila Meshcheryakova
+ */
 class ConverterTest extends PostServiceApplicationTests {
 
     @Test
-    void convert() {
-        Tag expected = Tag.builder()
+    void convertTagToTagResponse() {
+        Tag tag = Tag.builder()
```

Comment:
Please in test use your custom classes.
For example, if you change tag class that you need change this class too. It's bad practice.